sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -411,6 +411,96 @@ def test_indexing_tests(self):
         })
     self.assertEqual(result.failed, 0)
 
+  def test_groupby_tests(self):
+    result = doctests.testmod(
```

Comment:
1. Could you please add a docsting for `doctests.testmod()`?
For example, it's not clear what `use_beam` means and what parameters are allowed.
2. Do I understand correctly that this test will run the repl commands in pd.core.groupby.groupby (Pandas code)? If so, do we expect to be broken by Pandas adding new functionality that Beam Dataframe  doesn't support yet?

Reply:
1. Done.
2. You're right on both counts. That's actually why I recently switched our pandas dependency to restrict the next minor version (and made myself the owner of that dep so I'll know when we need to upgrade). In general bugfix releases shouldn't break this suite, but it's likely minor releases will, as they can add new functionality that we don't support.

Reply:
Thanks a lot!